src/main/steps/c100-rebuild/miam/no-need-with-reasons/content.ts

```diff
@@ -124,9 +147,24 @@ export const form: FormContent = {
 };
 
 export const generateContent: TranslationFn = content => {
-  const translations = languages[content.language]();
+  const translations = languages[content.language];
+  // const  reason  = prepareNoNeedWithReason(content.additionalData?.req.session.userCase || {},content.language);
   return {
     ...translations,
+    text: content.userCase?.miam_nonAttendanceReasons?.includes(MiamNonAttendReason.DOMESTIC)
```

Comment:
create a method in util called getExcemptionSummary that will return array of items like below,
[{
title: '',
reasons: [{
reason: '',
exemptions:['','']
}] 
}] and return exemptionSummary from generateContent.